airbyte-lib/airbyte_lib/_factories/connector_factories.py

```diff
@@ -3,24 +3,19 @@
 
 from typing import Any
 
-from airbyte_lib.cache import InMemoryCache
-from airbyte_lib.executor import Executor, PathExecutor, VenvExecutor
+from airbyte_lib._executor import Executor, PathExecutor, VenvExecutor
 from airbyte_lib.registry import get_connector_metadata
 from airbyte_lib.source import Source
 
 
-def get_in_memory_cache():
-    return InMemoryCache()
-
-
 def get_connector(
     name: str,
     version: str | None = None,
     pip_url: str | None = None,
     config: dict[str, Any] | None = None,
     use_local_install: bool = False,
-    install_if_missing: bool = True,
-):
+    install_if_missing: bool = False,
```

Comment:
Seems like this was a merge gone bad - could you revert that to `True` as default again?

Reply:
You are right. I think this got clobbered when resolving conflicts. Fixed now.